apps/admin/src/app/(api)/api/v1/token/route.ts

```diff
@@ -32,13 +32,16 @@ export const POST = async (req: NextRequest) => {
                 });
             }
 
-            if ((scriptData.needsScript && scriptData.script) || (data.needScript && scriptData.script)) {
+            const lastEntry = await getEnergyLastEntry(sensorData.id);
+
+            if ((sensorData.needsScript && sensorData.script) || (data.needScript && sensorData.script)) {
```

Comment:
```suggestion
            if ((sensorData.needsScript || data.needScript) && sensorData.script) {
```

Oder hat deine Formulierung noch eine besonderen Hintergedanken?